t4c/docker_entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
`set -euo pipefail` would be helpful so that the script fails on error.

Reply:
Addressed in #164